dev-python/pytools/pytools-2020.1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE='sqlite'
+
+inherit distutils-r1
+
+DESCRIPTION="Collection of tools missing from the Python standard library"
+HOMEPAGE="https://mathema.tician.de/software/pytools/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-python/appdirs-1.4.0[${PYTHON_USEDEP}]
+	>=dev-python/decorator-3.2.0[${PYTHON_USEDEP}]
+	>=dev-python/numpy-1.6.0[${PYTHON_USEDEP}]
+	>=dev-python/six-1.8.0[${PYTHON_USEDEP}]
+"
+DEPEND="
+	>=dev-python/setuptools-0.7.2[${PYTHON_USEDEP}]
+	test? (
+		${RDEPEND}
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/pytest-runner[${PYTHON_USEDEP}]
```

Comment:
Kill this.

Reply:
dropped this in favor of _distutils_enable_tests pytest_

Reply:
Actually, could you 'rebase' this file on the current version? It feels silly to copy the old version, then redo part of the changes that were done in it.

Reply:
What I understand is that you are not happy yet with this contribution. However I missed somehow what you would prefer instead. 
I understood you probably want one squashed/atomic commit (per package). Was this your point? Thanks for clarifying if you meant something different.

Reply:
one squashed/atomic commit (per package) is in tree now